### PR TITLE
New content for VB 2017

### PR DESCRIPTION
**Pull request by rpetrusha**
# New content for VB 2017

## Summary

Changed order of What's New items, added What's New for VB 2017, documented ref return values and added examples.

This PR replaces an earlier PR that I closed.

## Suggested Reviewers

@BillWagner @AnthonyDGreen

### CODE REVIEW

File: docs/visual-basic/getting-started/whats-new.md [hunk 1]
```diff
@@ -38,48 +37,104 @@ translation.priority.ht:
...
- Current version, see below  
+
+This topic lists key feature names for each version of Visual Basic, with detailed descriptions of the new and enhanced features in the lastest version of the language.
   
-## Current version  
- [Nameof](../../csharp/language-reference/keywords/nameof.md)  
+## Current Version
+
+Visual Basic / Visual Studio .NET 2017   
+For new features, see [Visual Basic 2017](visual-basic-2017)
```
Comment (BillWagner) [inline docs/visual-basic/getting-started/whats-new.md]: I think the link targets should start with a '#'. It doesn't appear to work right now.

File: docs/visual-basic/getting-started/whats-new.md [hunk 2]
```diff
@@ -38,48 +37,104 @@ translation.priority.ht:
...
+
+## Previous versions
+
+Visual Basic / Visual Studio .NET 2015   
+For new features, see [Visual Basic 14](visual-basic-14)
```
Comment (BillWagner) [inline docs/visual-basic/getting-started/whats-new.md]: Same think: link should start with '#'

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, this looks great.

The anchor links see incorrect. Once those are fixed, this should be ready to merge.

**Comment by rpetrusha** [comment]
@BillWagner, I've fixed the anchors, which now work.

**Review by mairaw — APPROVED** [review]
LGTM. Let's merge this!

**Pull request merged**